packages/expo-image/CHANGELOG.md

```diff
@@ -10,6 +10,12 @@
 
 ### 💡 Others
 
+## 1.2.3 — 2023-05-16
+
+### 🐛 Bug fixes
+
+- Upgrade SDWebImageAVIFCoder to fix compiling issue with libavif < 0.11.0. ([#22491](https://github.com/expo/expo/pull/22491) by [@matinzd](https://github.com/matinzd))
```

Comment:
Can you move this entry a few lines up, under `Unpublished / Bug fixes` category?